docs/index.md

```diff
@@ -4,86 +4,55 @@
 
-**Source Code**: <a href="https://github.com/dyvenia/viadot" target="_blank">https://github.com/dyvenia/viadot</a>
+**Source Code**: <a href="https://github.com/dyvenia/viadot/tree/2.0" target="_blank">https://github.com/dyvenia/viadot/tree/2.0</a>
 
 ---
 
 A simple data ingestion library to guide data flows from some places to other places.
 
-## Structure
-This documentation is following the diátaxis framework.
-
 ## Getting Data from a Source
 
-Viadot supports several API and RDBMS sources, private and public. Currently, we support the UK Carbon Intensity public API and base the examples on it.
+Viadot supports several API and RDBMS sources, private and public. An example of the use of the UK Carbon Intensity API.
 
 ```python
 from viadot.sources.uk_carbon_intensity import UKCarbonIntensity
 
 ukci = UKCarbonIntensity()
 ukci.query("/intensity")
 df = ukci.to_df()
-df
-```
 
-**Output:**  
+print(df)
+```
 
-|      | from              | to                |   forecast |   actual | index    |
-| ---: | ------------------|:------------------|-----------:|---------:|:---------|
-| 0    | 2021-08-10T11:00Z | 2021-08-10T11:30Z |        211 |      216 | moderate |
+**Output:**
 
+|      | from              | to                | forecast | actual | index    |
+| ---: | :---------------- | :---------------- | -------: | -----: | :------- |
+|    0 | 2021-08-10T11:00Z | 2021-08-10T11:30Z |      211 |    216 | moderate |
 
-The above `df` is a python pandas `DataFrame` object. The above df contains data downloaded from viadot from the Carbon Intensity UK API.
+The above `df` is a pandas `DataFrame` object. It contains data downloaded by `viadot` from the Carbon Intensity UK API.
 
 ## Loading Data to a Source
-Depending on the source, viadot provides different methods of uploading data. For instance, for SQL sources, this would be bulk inserts. For data lake sources, it would be a file upload. We also provide ready-made pipelines including data validation steps using Great Expectations.
-
-An example of loading data into SQLite from a pandas `DataFrame` using the `SQLiteInsert` Prefect task:
+Depending on the source, `viadot` provides different methods of uploading data. For instance, for SQL sources, this would be bulk inserts. For data lake sources, it would be a file upload. For ready-made pipelines including data validation steps using `dbt`, see [prefect-viadot](https://github.com/dyvenia/prefect-viadot).
 
-```python
-from viadot.tasks import SQLiteInsert
 
-insert_task = SQLiteInsert()
-insert_task.run(table_name=TABLE_NAME, dtypes=dtypes, db_path=database_path, df=df, if_exists="replace")
-```
+## Getting started
+### Prerequisites
+We assume that you have [Docker](https://www.docker.com/) installed.
 
+### Installation
+Clone the `2.0` branch, and set up and run the environment:
+  ```sh
+  git clone https://github.com/dyvenia/viadot.git -b 2.0 && \
+    cd viadot/docker && \
+    sh update.sh  && \
+    sh run.sh && \
+    cd ../
```

Comment:
This is incorrect as eg. `update.sh` will pull `viadot:latest` image by default; you'd need to use `sh update.sh -t 2.0-latest` to pull the correct one.

Reply:
[📝 Improved index.md](https://github.com/dyvenia/viadot/pull/657/commits/b0b0cf1528eb3eca265bc5a7ee2873a365d3686d)

Reply:
I removed the `update.sh` step because using `run.sh` downloads the latest image and creates a container and I think it is enough for typical user.

Reply:
That's not true -- up simply runs `docker compose up`; it doesn't pull the image (unless you run it for the first time and the image doesn't exist locally).